docs/samples/v1beta1/transformer/torchserve_image_transformer/README.md

```diff
@@ -142,3 +142,27 @@ Handling connection for 8080
 * Connection #0 to host localhost left intact
 {"predictions": [2]}
 ```
+
+
+
+## Testing with a local transformer & predictor in Kubernetes (developer guide)
```

Comment:
step by step how I tested

this was suggested to be moved to some other dev guide doc - which would be best? in the website repo?

Reply:
@yuzisun

Reply:
could also add how to test local kserve python sdk change in kubernetes, which is basically - 

(1) by copying your kserve/kserve/python/kserve repo into ./kserve  (where the Dockerfile is)
(2) change image to pip install your local code 
```
# RUN pip install --no-cache-dir kserve>=0.7.0rc0
RUN pip install --no-cache-dir -e .
RUN pip install kserve/kserve kserve
```

(3) build image & push to docker hub
(4) change image name in YAML 
(5) kubectl apply. 

would this be helpful to add to? I can make it more detailed if I put it in docs of course.

Reply:
after discussion - will add this to website docs.